tasks/activate-linux.yml

```diff
@@ -66,6 +74,7 @@
     msg: 
         - "Activate DSA when it's not activated or forecd to reactivate."
         - "force_reactivation: {{ force_reactivation }}"
+        - "dsa_control_argument: {{ dsa_control_argument }}"
```

Comment:
Please remove this as the argument might have security concern that we intent not to put it in msg.